### DISCUSSION TITLE
Input from BLE instead of VE.direct

### DISCUSSION THREAD

**Discussion by mflage** [iuriaranda/venus-docker#4] [Ideas]
Hi!

Have you looked at using BLE broadcast messages instead of VE.direct to get the metrics? For instance based on this: https://github.com/keshavdv/victron-ble ? Would that be doable?

--
Marius

**Comment by iuriaranda** [comment]
Hey! I've played around a bit with Victron BLE, and trying to manage an Orion DC/DC from an Arduino or Pi, but without much success so far. To be honest I haven't spent much time on it yet, but I plan to retake the project this summer. I didn't know that repo, it looks really interesting. I'll take a look at it. Thanks for the tip.

FYI I used the information from this other repo https://github.com/Olen/VictronConnect in my initial research of using BLE.

In any case, I don't think it has much to do with this repository (`venus-docker`), since Victron hasn't yet published a Venus package to manage their devices with Bluetooth. This repository mainly consists of being able to use some of the Venus OS functionality in Docker. If Victron ever publishes that functionality in Venus OS I'll make sure to integrate it here as well :slightly_smiling_face: 

If you don't mind, I'll convert the issue into a discussion, maybe we get some useful information out of it, that we and other people can use.
